wetterdienst/dwd/forecasts/stations.py

```diff
@@ -0,0 +1,49 @@
+from io import StringIO
+
+import pandas as pd
+import requests
+
+from wetterdienst.dwd.metadata.column_names import DWDMetaColumns
+
+MOSMIX_STATION_LIST = "https://www.dwd.de/DE/leistungen/met_verfahren_mosmix/" \
+                      "mosmix_stationskatalog.cfg?view=nasPublication"
```

Comment:
We should also follow this style elsewhere and put respective constants into the appropriate modules.